src/components/PageSection/index.tsx

```diff
@@ -15,20 +15,21 @@ interface BackgroundColorProps extends FlexProps {
   index: number
   background?: string
   hasZeroMargin?: boolean
+  padding?: string
```

Comment:
Can we extend this component with `<Box />`?

Reply:
Yea no need for padding being passed to it as a prop - passed `p = getPadding()` to the component instead. It needs some of the flex properties though, so extended using that.